gui/wxpython/gmodeler/model.py

```diff
@@ -2883,6 +2883,7 @@ def _writePythonAction(self, item, variables={}, intermediates=None):
         for param in item.GetParams()["params"]:
             value = param["value"]
             age = param["age"]
+            overwrite = self.model.GetProperties().get("overwrite", "False")
```

Comment:
@pesekon2 To be consistent with other commands, `overwrite` should be defined if overwrite option is enabled in the model.

Reply:
Fixed in [3aabc6f](https://github.com/OSGeo/grass/pull/1919/commits/3aabc6f9c5abb97822c34bbc79122be2ef9eb898)